tests/udp_socket.rs

```diff
@@ -140,6 +140,92 @@ fn get_ttl_without_previous_set() {
     socket1.ttl().expect("unable to get TTL for UDP socket");
 }
 
+#[test]
+fn set_get_broadcast() {
+    let socket1 = UdpSocket::bind(any_local_address()).unwrap();
+
+    socket1.set_broadcast(true).unwrap();
+    assert_eq!(socket1.broadcast().unwrap(), true);
+
+    socket1.set_broadcast(false).unwrap();
+    assert_eq!(socket1.broadcast().unwrap(), false);
+
+    assert!(socket1.take_error().unwrap().is_none());
+}
+
+#[test]
+fn get_broadcast_without_previous_set() {
+    let socket1 = UdpSocket::bind(any_local_address()).unwrap();
+
+    socket1
```

Comment:
Want assert its false?

Reply:
No, just want to call the get w/o previous set. Plus, it's value might be platform dependent, and we'd have to check value based on platform.

Reply:
Fair enough.